lib/wovnrb/services/html_converter.rb

```diff
@@ -142,7 +142,7 @@ def insert_hreflang_tags
     # Remove wovn snippet code from dom
     def strip_snippet
       @dom.xpath('//script').each do |script_node|
-        script_node.remove if script_node['src'] && script_node['src'] =~ /^\/\/j.(dev-)?wovn.io(:3000)?\//
+        script_node.remove if script_node['src'] && !script_node['data-wovnio'].blank? || [@store.settings['api_url_versioned_widget'], 'j.wovn.io'].any? { |url| script_node['src'].include? url }
```

Comment:
looks good, although the array could be moved to the constructor I think. (like `@widget_urls = [...]`

Reply:
I added it as a method, changing the constructor seemed a bit excessive.